### PR TITLE
lowdown: run pre/postConfigure hooks

### PR DESCRIPTION
**Pull request by sternenseemann**
###### Motivation for this change

Make overriding work as expected.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/typesetting/lowdown/default.nix [hunk 1]
```diff
@@ -15,10 +15,12 @@ stdenv.mkDerivation rec {
     ++ lib.optionals stdenv.isDarwin [ fixDarwinDylibNames ];
 
   configurePhase = ''
+    runHook preConfigure
     ./configure PREFIX=''${!outputDev} \
```
Comment (SuperSandro2000) [inline pkgs/tools/typesetting/lowdown/default.nix:19]: Can we use configureFlags here instead?

Comment (sternenseemann) [reply pkgs/tools/typesetting/lowdown/default.nix:19]: No, `oconfigure` doesn't like the flags `mkDerivation` always sets which is why used a manual `configurePhase` in the first place.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112524` at 798f9f2a run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/4c4c14201e31000a162ba87ea508de0c">hydra-unstable</a></li></li>
    <li><a href="https://gist.github.com/ee95a815ba96c949f9248d5cf8a6c58c">lowdown</a></li></li>
    <li><a href="https://gist.github.com/99e5bea235cd92ce53516b841b5aeb1a">nix-update</a></li></li>
    <li><a href="https://gist.github.com/1e2ebb6c0df5911e8511aef6e1846264">nixFlakes</a></li> (nixUnstable)</li>
    <li><a href="https://gist.github.com/d5e3cd6a99c6b6b3cfcc584d6e6cc623">nixpkgs-review</a></li></li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112524` at 798f9f2a run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/c425d6b63e6d13986b31fe32eb5a801b">hydra-unstable</a></li></li>
    <li><a href="https://gist.github.com/0a9bd692fbafdb566bee578c56c3983e">lowdown</a></li></li>
    <li><a href="https://gist.github.com/911b290c0b0fd641383fdc5e4c3fbc3e">nix-update</a></li></li>
    <li><a href="https://gist.github.com/0cbd561c55da15cd4841128a3afc271c">nixFlakes</a></li> (nixUnstable)</li>
    <li><a href="https://gist.github.com/036eb16c0c99cd843c6dccd20eda9cdd">nixpkgs-review</a></li></li>
  </ul>
</details>

**Review by bjornfor — APPROVED** [review]

**Pull request merged**